float32[] waypoint
---
bool success
